### DISCUSSION TITLE
Gajillions of genetic material in Alien_Workshop module

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14887] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Alien_Workshop module spawns a ton of genetic material

Those little storage vats are everywhere and spawn from 2 to 4 UGM every time.

Might want to tag them as "lessergeneticstorage" and reduce amount/probabilities of spawning UGM there?

![image](https://github.com/user-attachments/assets/295c8f79-689f-40d8-b3cf-83633f016706)
![image](https://github.com/user-attachments/assets/b218e200-b5de-451d-b893-a7c0075e7492)
![image](https://github.com/user-attachments/assets/cce3cbca-5ded-43f5-ad28-2a971158d451)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/pull/5915 (PR in our private repo), we're planning on pushing a hotfix to this soon. Sorry for the inconvenience (or maybe too much convenience :D)!
